pkg/cloudprovider/nodecontroller/nodecontroller.go

```diff
@@ -253,6 +257,12 @@ func (nc *NodeController) Run(period time.Duration, syncNodeList bool) {
 			glog.Errorf("Error monitoring node status: %v", err)
 		}
 	}, nc.nodeMonitorPeriod)
+
+	go util.Forever(func() {
```

Comment:
please explain why you need a goroutine/queue (as opposed to, say, listing pods and just not doing the CanAccept() if a bad node has no pods or something)?